From e4798c613e08485398eb3b9944a2654413e3fc5c Mon Sep 17 00:00:00 2001
From: Swee Yee Fonn <swee.yee.fonn@intel.com>
Date: Wed, 14 Oct 2020 10:53:08 +0800
Subject: [PATCH] Update setup host script for wmi device

Add additional setup for support
HP wmi device

Change-Id: Iacd0f57944f7cba5dc8d0895d9744e6678b774d9
Tracked-On: OAM-94299
Signed-off-by: Swee Yee Fonn <swee.yee.fonn@intel.com>

diff --git a/scripts/setup_host.sh b/scripts/setup_host.sh
index 8d77773..028f078 100755
--- a/scripts/setup_host.sh
+++ b/scripts/setup_host.sh
@@ -28,7 +28,7 @@ function ubu_changes_require(){
     if [ x$res = xn ]; then
         return 1
     fi
-    sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq acpica-tools acpid
+    sudo apt install -y wget mtools ovmf dmidecode python3-usb python3-pyudev pulseaudio jq acpica-tools acpid bash

     # Install libs for vm-manager
     sudo apt install -y libglib2.0-dev libncurses-dev libuuid1 uuid-dev libjson-c-dev
--
2.29.2

